sql/core/src/test/scala/org/apache/spark/sql/errors/QueryExecutionErrorsSuite.scala

```diff
@@ -653,26 +653,6 @@ class QueryExecutionErrorsSuite
         "alternative" -> "",
         "config" -> SQLConf.ANSI_ENABLED.key))
   }
-
-  test("CAST_OVERFLOW: from long to ANSI intervals") {
-    Seq(
-      LongType -> "9223372036854775807L",
-      DecimalType(19, 0) -> "9223372036854775807BD").foreach { case (sourceType, sourceValue) =>
-      Seq("INTERVAL YEAR TO MONTH", "INTERVAL HOUR TO MINUTE").foreach { it =>
-        checkError(
-          exception = intercept[SparkArithmeticException] {
-            sql(s"select CAST($sourceValue AS $it)").collect()
-          },
-          errorClass = "CAST_OVERFLOW",
```

Comment:
Why do you remove this check for `CAST_OVERFLOW`?

Reply:
It must have got removed during merge. Sorry for the unintentional mistake. Added the check.